modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/StoriesDataSource.kt

```diff
@@ -4,5 +4,8 @@ import au.com.shiftyjelly.pocketcasts.endofyear.stories.Story
 import kotlinx.coroutines.flow.Flow
 
 interface StoriesDataSource {
+    suspend fun isEligibleForStories(): Flow<Boolean>
+    suspend fun hasFullListeningHistory(): Flow<Boolean>
+    suspend fun syncListeningHistory(): Flow<Boolean>
```

Comment:
Functions that return `Flow` shouldn't be suspending. Suspending mechanism is already built into `Flow`. Any `suspend fun foo(): Flow<T>` can be expressed as `fun foo(): Flow<T>`. This change should be propagated all the way downstream to view models and wherever else these functions are used directly or indirectly.

Reply:
Removed in 35f1d40